test/unit/org/apache/cassandra/cql3/validation/entities/UFJavaTest.java

```diff
@@ -796,4 +801,55 @@ public void testAllNativeTypes() throws Throwable
                            "AS 'return 0;'");
         }
     }
+
+    @Test
+    public void testUDFToCqlString()
+    {
+        UDFunction function = UDFunction.create(new FunctionName("my_ks", "my_function"),
+                                                Arrays.asList(ColumnIdentifier.getInterned("column", false)),
+                                                Arrays.asList(UTF8Type.instance),
+                                                Int32Type.instance,
+                                                false,
+                                                "java",
+                                                "return 0;");
+
+        Assert.assertTrue(function.toCqlString(true, true).contains("CREATE FUNCTION IF NOT EXISTS"));
+        Assert.assertFalse(function.toCqlString(true, false).contains("CREATE FUNCTION IF NOT EXISTS"));
```

Comment:
Nice tests. Not related to the changes introduced in this ticket but, since we are adding a specific test for the `toCqlString` method for the first time, we could also easily check the remaining argument for the sake of completeness. The argument is not used, so we could just verify that it doesn't have any effect:
```
Assert.assertEquals(function.toCqlString(true, true), function.toCqlString(false, true));
Assert.assertEquals(function.toCqlString(true, false), function.toCqlString(false, false));
```
The same could be done in `testUDAToCqlString`. Feel free to ignore if you think it doesn't add any value.

Reply:
added thanks